app/views/users/_profile.html.erb

```diff
@@ -157,14 +157,18 @@
                               aria: { describedby: "#{field.attribute_name}-description" },
                               data: { character_span_id: "#{field.attribute_name}-characters" },
                               class: "crayons-textfield") %>
-            <div id="<%= field.attribute_name %>-description" class="self-end fs-s">
-              <span class="screen-reader-only">Available characters used: </span>
-              <span id="<%= field.attribute_name %>-characters"></span>/<%= character_count_denominator(field["input_type"]) %>
+            <div style="display: grid; grid-template-columns: 85% 15%;">
+              <div>
```

Comment:
This containing `<div>` is necessary for the `grid` to have an object to act upon. Without it, the grid collapses and the character count simply shifts all the way to the left.